stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/MessageListView.kt

```diff
@@ -592,7 +593,14 @@ public class MessageListView : ConstraintLayout {
     }
     private val defaultOnPollOptionClickListener = OnPollOptionClickListener { _, _, _ -> false }
     private val defaultOnPollCloseClickListener = OnPollCloseClickListener { false }
-    private val defaultOnViewPollResultClickListener = OnViewPollResultClickListener { false }
+    private val defaultOnViewPollResultClickListener = OnViewPollResultClickListener { poll ->
+        val fm = context.getFragmentManager() ?: return@OnViewPollResultClickListener false
```

Comment:
We should avoid a `return` in the elvis operator as its unreadable. Also why not wrap this in `try-catch` as a defensive measure since we are accessing the `context` here.

Something in the lines of
```kotlin
private val defaultOnViewPollResultClickListener = OnViewPollResultClickListener { poll ->
    try {
        val fragmentManager = context.getFragmentManager()
        val viewShown = fragmentManager?.let { nonNullFragmentManager ->
            PollResultsDialogFragment.newInstance(poll)
                .show(nonNullFragmentManager, PollResultsDialogFragment.TAG)
            true
        } ?: false
        viewShown
    } catch (e: Exception) {
        // Log the exception if needed
        false
    }
}
```